tests/pr_test.go

```diff
@@ -25,7 +37,8 @@ func setupOptions(t *testing.T, prefix string) *testhelper.TestOptions {
 
 	// query for best zone to deploy powervs example, based on current connection count
 	// NOTE: this is why we do not want to run multiple tests in parallel
-	options.Region, _ = testhelper.GetBestPowerSystemsRegion(options.RequiredEnvironmentVars["TF_VAR_ibmcloud_api_key"], options.BestRegionYAMLPath, options.DefaultRegion)
+	options.Region, _ = testhelper.GetBestPowerSystemsRegionO(options.RequiredEnvironmentVars["TF_VAR_ibmcloud_api_key"], options.BestRegionYAMLPath, options.DefaultRegion,
+		testhelper.TesthelperTerraformOptions{CloudInfoService: sharedInfoSvc, ExcludeActivityTrackerRegions: true})
```

Comment:
Also I just noticed that you set ExcludeActivityTrackerRegions to "true", but for your case I'm not sure you need that. What that will do is eliminate any regions for selection if that region already has an active ActivityTracker, you probably don't need that restriction. You can either set it false or just remove it (default is false)

Reply:
ok, I will remove the ExcludeActivityTrackerRegions.